src/appleseed.studio/python/pythoninterpreter.h

```diff
@@ -32,6 +32,8 @@
 // appleseed.foundation headers.
 #include "foundation/core/concepts/noncopyable.h"
 
+#include "mainwindow/mainwindow.h"
```

Comment:
that way I need to include mainwindow.h in module.cpp and in every file where I use PythonInterpreter and get MainWindow from it. Is it ok?